client/README.md

```diff
@@ -1,208 +1,389 @@
-client package.
-
-## Supported Python Versions 
-Python >= 3.7
-
-## Installation
-
-Clone or download this repository and install the exemplary Python client: 
-
+# BigQuery Migration Service Batch SQL Translator CLI
+
+The [BigQuery Migration Service][BQMS] includes a product known as the
+[batch SQL translator][batch SQL translator] that supports translation of a 
+wide variety of SQL dialects to Google Standard SQL. This command line tool
+simplifies the process of using the batch SQL translator by providing a
+framework for pre and postprocessing of SQL files as well as upload/download of
+SQL files to/from GCS.
+
+- [Quickstart](#quickstart)
+- [Installation](#installation)
+- [Basic Usage](#basic-usage)
+  - [Environment Variables](#environment-variables)
+  - [config.yaml](#configyaml)
+  - [Recommended Usage](#recommended-usage)
+- [Metadata Lookup](#metadata-lookup)
+    - [DDL and Metadata Dump](#ddl-and-metadata-dump)
+    - [Unqualified References](#unqualified-references)
+- [Pre and Postprocessing](#pre-and-postprocessing)
+  - [Handling Macro/Templating Languages](#handling-macrotemplating-languages)
+  - [Custom Pre and Postprocessing Logic](#custom-pre-and-postprocessing-logic)
+- [Renaming SQL Identifiers](#renaming-sql-identifiers)
+- [Deploying to Cloud Run](#deploying-to-cloud-run)
+
+## Quickstart
+
+```shell
+# Install the CLI as described in the Installation section below.
+
+# Copy the example project directory to a project directory of your own.
+cp -R examples/teradata/sql <YOUR_PROJECT_DIRECTORY>
+
+# Change directory to your project directory.
+cd <YOUR_PROJECT_DIRECTORY>
+
+# Remove the example input files from the input directory.
+rm -rf input/*
+
+# Copy the files you would like to translate into the input directory.
+cp <YOUR_INPUT_FILES> input/
+
+# Edit the config/config.yaml file appropriately as described in the Basic Usage
+# section below.
+nano config/config.yaml
+
+# Set the appropriate environment variables as described in the Basic Usage
+# section below.
+export BQMS_VERBOSE="False"
+export BQMS_MULTITHREADED="True"
+export BQMS_PROJECT="<YOUR_GCP_PROJECT>"
+export BQMS_GCS_BUCKET="<YOUR_GCS_BUCKET>"
+
+# Run the translation.
+./run.sh
 ```
-pip3 install dwh-migration-tools/client
-```
-
-Install the gcloud CLI following the [instructions](http://cloud.google.com/sdk/docs/install).
-
-### [Optional] gcloud login and authentication
 
-The program will first validate the login and credential status of
-gcloud. If the validation steps failed, the program will run the following two
-commands automatically and bring up a page on browser for your agreement of
-using your Google account.
+## Installation
 
-However, it's recommended that you run these two commands on the terminal first as a one-time setup requirement:
+Prerequisites: Python >= 3.7.2.
 
-Log in to gcloud:
+Preferred OS: Linux or MacOS.
 
-```
-gcloud auth login
+```shell
```

Comment:
We don't need to download this repo first?
Also which directory we need to be to run this command successfully?

Reply:
> We don't need to download this repo first?

Yes, the repo still needs to be cloned I just omitted that step.

> Also which directory we need to be to run this command successfully?

Everything is written with the assumption you are in the same directory as the `README`.

I can show the `git clone ... && cd ..` to make it more clear.